src/styles/global.css

```
@@ -255,35 +254,49 @@ p {
+  transition: opacity var(--link-animation-duration) var(--ease-out);
 }
 
 a:not(.category-link)::after {
   content: '';
   position: absolute;
   bottom: -2px;
   left: 0;
-  width: 0;
+  width: 100%;
   height: 1px;
   background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)